### DISCUSSION TITLE
Always start application on a specific monitor (_not_ workspace)

### DISCUSSION THREAD

**Discussion by SimonBrandner** [i3/i3#5330] [Q&A] [answered]
Hi all!

I would like a certain app to be always started on a specific monitor. So far all suggestions seem to assign the window class to a specific workspace, which is sadly not what I want. Specifically, I am talking about KRunner: I am using i3wm in combination with KDE Plasma and KRunner isn't behaving the best, so I would like to make it always appear on the main monitor.

Is there a solution to this?

Thanks!

**Comment by orestisfl** [comment, marked as answer]
See https://i3wm.org/docs/userguide.html#assign_workspace

```

# Assign urxvt to the output right of the current one
assign [class="^URxvt$"] → output right

# Assign urxvt to the primary output
assign [class="^URxvt$"] → output primary
```

**Reply by SimonBrandner** [reply]
Thank you, that did the trick! I've completely missed it in the docs :man_facepalming:
